client/README.md

```diff
@@ -15,7 +15,7 @@ Requires node 10 & npm 6 or higher, and a running [api server](../server/README.
 
 ## Develop App Locally
 
-Set API server:
+### Run API server
 
```

Comment:
To get the API server running, follow the steps documented for the [api server](https://github.com/GoogleCloudPlatform/avocano/server/README.md#local-dev) and then make sure you set up the environment variable API_URL: